sylt-parser/src/parser.rs

```diff
@@ -119,6 +119,12 @@ impl PartialEq for Identifier {
 #[derive(Debug, Clone, PartialEq)]
 pub enum AssignableKind {
     Read(Identifier),
+    /// Instance of an Enum variant `Abc.A(1, 2, 3)`
```

Comment:
Is this really how they look? I thought there was a space.

Reply:
It's syntactially correct - but the formatter writes them with a space.